### PR TITLE
[Feat:] Added Back to Home Page functionality

### PR DESCRIPTION
**Pull request by RitochitGhosh**
## Description
Provided a back button functionality in the sidebar of dashboard, which redirects to home page

### Method Adopted:
- Modified the src/app/dashboard/page.jsx file.
- Used a Link to direct from Dashboard page to Home page. 
- Used "ArrowBackIosNewRoundedIcon" from Material UI as a displaying Icon. 

### Here is how it looks now:
Screenshot 2024-05-20 123814

### CODE REVIEW

File: src/app/dashboard/page.jsx [hunk 1]
```diff
@@ -7,6 +7,10 @@ import FormGroup from "@mui/material/FormGroup";
 import FormControlLabel from "@mui/material/FormControlLabel";
 import AddIcon from "@mui/icons-material/Add";
 import { redirect } from "next/navigation";
+import Link from "next/link";
+import HomeIcon from '@mui/icons-material/Home';
```
Comment (Arindam200) [inline src/app/dashboard/page.jsx:11]: This is not used anywhere.

Comment (Arindam200) [inline src/app/dashboard/page.jsx:11]:
```suggestion

```

**Review by Swpn0neel — COMMENTED** [review]
Unnecessary importing of a Home Icon from Material UI, please remove it!!

**Comment by Arindam200** [comment]
Also fix the merge conflicts

**Pull request closed without merging**